recipes/asimov/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "asimov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 160df0c977cf5c65794f244957c7f4436d250bbf63fd944654219bf29b9114e3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - locutus=asimov.locutus:cli
+    - olivaw=asimov.olivaw:olivaw
+    - asimov=asimov.olivaw:olivaw
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools
+    - setuptools_scm
+  run:
+    - click
+    - flask
+    - git
+    - gitpython
+    - gwpy
+    - igwn-auth-utils
+    - ligo-gracedb
+    - liquidpy
+    - lscsoft-glue
+    - networkx
+    - numpy
+    - otter-report >=0.3.3
+    - python >=3.7
+    - python-gitlab
+    - python-htcondor
+    - pytz
+    - pyyaml
+    - requests
+    - tinydb
+
+test:
+  requires:
+    - pip
+    - git
+  imports:
+    - asimov
+  commands:
+    # check requirements
+    - python -m pip check
+    # check entry points
+    - locutus --help
+    - asimov --help
+    - olivaw --help
+
+about:
+  home: https://asimov.docs.ligo.org/asimov/
+  doc_url: https://asimov.docs.ligo.org/asimov/
+  dev_url: https://git.ligo.org/asimov/asimov.git
+  license: ISC
```

Comment:
You report MIT upstream: https://git.ligo.org/asimov/asimov/-/blob/review/setup.py#L38
Doesn't affect this package but should maybe be changed at some point.

Reply:
Oh, good spot, thanks!